FindOpenGLES2.cmake

```diff
@@ -0,0 +1,36 @@
+# - Try to find OpenGLES2
+# Once done this will define
+#
+#  OPENGLES2_FOUND        - system has OpenGLES2
+#  OPENGLES2_INCLUDE_DIR  - the GLES2 include directory
+#  OPENGLES2_LIBRARIES    - Link these to use OpenGLES2
+
+find_package(PkgConfig)
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(OpenGLES2 glesv2)
+endif(PKG_CONFIG_FOUND)
+
+if(NOT OPENGLES2_FOUND)
```

Comment:
IMO this is wrong, pkg config info should be only treated as hint, but not as the truth.

Reply:
Yes, but I would remove everything not needed from it (gles3, HAS_GLES define)

Reply:
I add also the FindOpenGl.cmake and removing the "add_definitions(-DHAS_GLES2)" and "add_definitions(-DHAS_GL)" from CMakeLists.txt. And changing the "#include" structure of the shader Class Files.

This from Kodi are more correct fro me.